application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsDashboardSheet.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="Analytics.Code.AnalyticsDashboardSheet" locale="">
+  <web>Analytics.Code</web>
+  <name>AnalyticsDashboardSheet</name>
+  <language/>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>0</translation>
+  <creator>XWiki.Admin</creator>
+  <creationDate>1689589838000</creationDate>
+  <parent>Analytics.Code.AnalyticsDashboardClass</parent>
+  <author>XWiki.Admin</author>
+  <contentAuthor>XWiki.Admin</contentAuthor>
+  <date>1689589838000</date>
+  <contentUpdateDate>1689589838000</contentUpdateDate>
+  <version>1.1</version>
+  <title>#if($doc.documentReference.name == 'AnalyticsDashboardSheet')AnalyticsDashboard Sheet#{else}$services.display.title($doc, {'displayerHint': 'default', 'outputSyntaxId': 'plain/1.0'})#end</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
```

Comment:
All code pages should be hidden. Did you applied `xar:format`? It should take care of this and other properties I see you have on your pages (e.g. contentUpdateDate, the `&#xd;` at the end of line)